### PR TITLE
Blog App - Views

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone 6: Views
In this PR, I completed the following steps:

- Implement the design from the sneak peek wireframes.
- Add some extra styles
- Add navigation between pages

ℹ️ To run this project:

- Clone this repository to your local machine:

```sh

  git clone git@github.com:anthonymr/anthony-blog.git
  cd anthony-blog
```
- Install dependencies:
```sh
  bundle install
```
- Recreate Data Base:
```sh
  rake db:schema:load
```
Run `Rails` server:

```  sh
  rails s  
```

### CODE REVIEW

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <div class="container">
+    <div><%= render partial: 'shared/comment', collection: post.comments %></div>
+  </div>
+<% end %>
+
+<% if local_assigns[:show_lasts_comments] && Post.last_comments(post).any? %>
+  <div class="container">
+    <div><%= render partial: 'shared/comment', collection: Post.last_comments(post) %></div>
+  </div>
+<% end %>
```
Comment (Gambit142) [inline app/views/posts/_post.html.erb:33]: - [ ] Great work using partials. They make it easy to render fractions of pages that reoccur on different pages. Great work champ 👏. Great work using the ruby on rails helper `local_assigns`. You used the helper properly 👍

**Review by Gambit142 — APPROVED** [review]
Hi @anthonymr 👋,

## Status: Approved 🟢   

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Created index and show pages for User and Post models ✔️ 
- Pages created were identical to the wireframe provided ✔️ 
- Great use of partials ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_post.html.erb (hunk 1 above)
Comment (anthonymr) [reply app/views/posts/_post.html.erb:33]: Thanks very much, I tried My Best...

Comment (Gambit142) [reply app/views/posts/_post.html.erb:33]: @anthonymr, you did. Great work! 👏

**Pull request merged**